backend/src/main/resources/application.properties

```diff
@@ -1,17 +1,30 @@
+#   local dev values
+DB_HOST=localhost
+DB_PORT=3306
+DB_NAME=skynest
+DB_USERNAME=user
+DB_PASSWORD=123456
```

Comment:
Good approach for this variables and values is to have separate application.properties for different Spring profiles :) The files would be application-{profile}.properties where you set just the values for keys that are overridden in corresponding profile. Again create a technical dept task in backlog, you don't need to change it now

Reply:
It's a task now [SJQ2T-59](https://htecgroup.atlassian.net/browse/SJQ2T-59).